x/blob/README.md

```diff
@@ -2,44 +2,141 @@
 
 ## Abstract
 
-The `x/blob` module enables users to pay for arbitrary data to be published to the Celestia blockchain. Users create a single `BlobTx` that is composed of:
+The `x/blob` module enables users to pay for arbitrary data to be published to
+the Celestia blockchain. Users create a single `BlobTx` that is composed of:
 
-1. Multiple `Blob`s (Binary Large OBjects): the data they wish to publish. A single `Blob` is composed of:
+1. Multiple `Blob`s (Binary Large OBjects): the data they wish to publish. A
+   single `Blob` is composed of:
     1. `NamespaceId  []byte`: the namespace this blob should be published to.
     1. `Data         []byte`: the data to be published.
-    1. `ShareVersion uint32`: the version of the share format used to encode this blob into a share.
+    1. `ShareVersion uint32`: the version of the share format used to encode
```

Comment:
I don't think we have one yet although I definitely agree that anyplace we mention some versioning we should be clear about what constitutes a version change and especially for this specific parameter because it's user facing

Reply:
added a link to ADR014, and mentioned that we specify there how it effects the share encoding and when that value would get incrememnted